docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,231 @@
+
+### Requirements
+ 
+* You must have a VPC, with NAT gateways on private subnets. NAT gateway is necessary to enable internet access for Kubernetes node group subnets.
+* You must have at least 2 subnets in different AZs for the Kubernetes cluster. This is mandatory to create an EKS cluster.
+* You must have at least one private subnet for the managed database other than Dynamo DB.
+* You must create subnets with the prefix at least _/24_ for the Kubernetes cluster to work without issues even after scaling. 
+
+### Recommendations
+
+* Kubernetes should be private in production and should provide secure access with SSH or VPN. You can use a bastion server as a host machine for secure access to the private network.
+* Subnets should be in multiple availability zones, to enable fault tolerance for the production use.
+* Kubernetes cluster should be public to enable the envoy public endpoint(Scalar DL can be accessed from the internet). The Kubernetes cluster in public is not recommended for production.
+
+_Tip_ 
+
+_If you plan to create three or more nodes in a Kubernetes node group for high availability, create at least 3 subnets in different AZs for the Kubernetes cluster.__
+
+[Create an Amazon VPC](https://docs.aws.amazon.com/eks/latest/userguide/create-public-private-vpc.html) with the above requirements and recommendations along with your organizational or application standards.
+
+## Step 2. Set up a database 
+
+In this section, you will set up a database for Scalar DL.
+
+### Requirements
+
+* You must have a database that Scalar DL supports.
+
+Follow the [Set up a Scalar DL supported database](ScalarDLSupportedDatabase.md) document to Set up a Database for Scalar DL.
+
+## Step 3. Configure EKS
+
+Amazon EKS provides a scalable and highly-available Kubernetes control plane that runs across multiple AWS availability zones, which helps you to achieve the high availability and scalability for Scalar DL. 
+This section shows how to configure a Kubernetes service for Scalar DL deployment.
+
+### Prerequisites
+
+Install the following tools on your host machine:
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html#cliv2-linux-install): In this guide, AWS CLI is used to create a kubeconfig file to access the EKS cluster.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/): Kubernetes command-line tool to manage EKS cluster. Kubectl 1.19 or higher is required.
+
+### Requirements
+
+* You must have an EKS cluster with Kubernetes version 1.19 or above in order to use our most up-to-date configuration files.
+* Kubernetes node group must be labeled with key as `agentpool` and value as `scalardlpool` for [ledger](https://github.com/scalar-labs/scalardl) and [envoy](https://www.envoyproxy.io/) deployment.
+
+### Recommendations
+
+* Kubernetes node size should be `m5.xlarge` for deploying `ledger` and `envoy` pods, because each node requires 4 vCPUs and 16 GiB of memory for deploying ledger and envoy pods.
```

Comment:
`m5.large` is used by default in `scalar-terraform`. Have you tried to deploy with cloudwatch logging disabled?
https://github.com/scalar-labs/scalar-terraform/blob/0dc137dad849e0802edfa4063c89cdd93898d12a/modules/aws/kubernetes/locals.tf#L46

Reply:
> Have you tried to deploy with cloudwatch logging disabled?
Yes. When `m5.large` is used, either cloudwatch agent or logging can only be used. One will fail when both are deployed.